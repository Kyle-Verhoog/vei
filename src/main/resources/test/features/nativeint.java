public class Nativeint {
    public static native int m(int i);
}