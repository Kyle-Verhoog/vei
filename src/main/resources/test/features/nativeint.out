PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
STATIC ┃ static
NATIVE ┃ native
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ i
) ┃ )
; ┃ ;
} ┃ }
